src/typescript/frontend/src/components/pages/emojicoin/components/chat/components/message-container/index.tsx

```diff
@@ -44,7 +44,10 @@ const MessageContainer: React.FC<MessageContainerProps> = ({ message }) => {
               href={toExplorerLink({ value: message.version, linkType: "version" })}
             >
               <Text textScale="pixelHeading4" color="lightGray" textTransform="uppercase">
-                {truncateAddress(message.sender)}
+                {/* Note: someone could have a name starting with "0x", is it a problem ? */}
```

Comment:
Good point, I think you could add a flag to the data passed in (`message` prop I think...? Add a `ansName?: boolean` field) to indicate its an ANS name and truncate it based on that

Reply:
I thought about that, but I wondered if it was useful or not.

Let's say my name is of the form 0x..., I probably want it formatted as an address, right ? Thought on this ?

Reply:
Yes I was thinking about this as well, I actually think you're right. If it's in the form of an address just truncate it as such.

You can just call `AccountAddress.isValid` with `strict: true` to check if the address is a valid account address with the `ts-sdk` for this